api/src/main/java/org/eclipse/microprofile/reactive/messaging/Channel.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2018-2019 Contributors to the Eclipse Foundation
```

Comment:
Not sure we should handle this in this PR, but we are in 2020.

Reply:
I forgot we are in 2020. I am behind:o

Reply:
fixed